docker/www/script/entry.sh

```diff
@@ -22,6 +22,13 @@ if [ "$1" == '' ]; then
 	# Make sure Apache shuts down properly
 	trap stop SIGINT SIGTERM
 
+cd /tmp || return
+tar -xf files.tar.gz
+rm -rf /var/www/html/sites/default/files/*
```

Comment:
fjern mappen files i stedet for filerne. Så er du sikker på at få alt ukrudt med :-)